src/api/app/views/webui/shared/_editor_modal.html.haml

```diff
@@ -3,8 +3,7 @@
     .modal-content
       .modal-header
         %h5.modal-title Editor settings
-        %button.close{ 'data-dismiss': 'modal' }
-          %span ×
+        %button.btn.btn-close.float-end{ 'data-dismiss': 'modal' }
```

Comment:
```suggestion
        %button.btn.btn-close.float-end{ type: 'button', 'data-dismiss': 'modal' }
```
We could add the `type: 'button'` part, here and in a couple of places more where is missing.